src/routes/api.js

```diff
@@ -36,4 +37,6 @@ module.exports = function (app, middleware, controllers) {
 	router.post('/topic/thumb/upload', middlewares, uploadsController.uploadThumb);
 
 	router.post('/user/:userslug/uploadpicture', middlewares.concat([middleware.exposeUid, middleware.authenticate, middleware.canViewUsers, middleware.checkAccountPermissions]), controllers.accounts.edit.uploadPicture);
+
+	helpers.setupApiRoute(router, 'get', '/user/invite-groups', [middleware.authenticate], controllers.user.getInviteGroups);
```

Comment:
Let's move this to `src/routes/write/users.js`, `GET /api/v3/users/<uid>/inviteGroups`

Reply:
This is used as a self-consumed endpoint, so {uid} route param seemed redundant. Change has been made nevertheless. :)